README.rst

```diff
@@ -19,3 +19,8 @@ high-degree of transparency.
 ZODB is an ACID Transactional database.
 
 To learn more, visit: http://www.zodb.org
+
+The githib repository is: at https://github.com/zopefoundation/zodb
+
+If you're interested in contributing to ZODB itself, see the
+`developer notes <DEVELOPERS.rst>`_.
```

Comment:
Although that renders just fine locally in `restview --long-description`, is that going to be a broken link on PyPI? I didn't know it would automatically render more than long-description.